ext/standard/tests/file/proc_open_with_wrong_resource_type.phpt

```diff
@@ -0,0 +1,13 @@
+--TEST--
+proc_open does not leak memory when called with wrong resource type in descriptorspec
+--FILE--
+<?php
+    $context = stream_context_create();
+    $child   = proc_open('not_a_real_command_but_I_dont_care', array(0 => $context), $pipes);
+    echo "Not reached\n";
```

Comment:
Two style nits: Prefer `?>` at end of file in tests, because it makes the phpt file directly executable. For exceptions, prefer to explicitly catch them with

```
try {
    ...
} catch (TypeError $e) {
    echo $e->getMessage(), "\n";
}
```

Reply:
Updating.